lib/puppet/provider/group/windows_adsi.rb

```diff
@@ -33,6 +33,7 @@ def members_insync?(current, should)
     if @resource[:auth_membership]
       current_users == specified_users
     else
+      return true if specified_users.empty?
```

Comment:
It would need specified like this: `elsif specified_users.empty? && !@resource[:auth_membership]`